modeltranslation/tests/translation.py

```diff
@@ -85,6 +85,26 @@ class ForeignKeyModelTranslationOptions(TranslationOptions):
     )
 
 
+@register(models.SetNullForeignKeyBaseModel)
+class SetNullForeignKeyBaseModelTranslationOptions(TranslationOptions):
+    fields = [
+        "test",
+    ]
+
+@register(models.FirstSetNullForeignKeyModel)
+class FirstSetNullForeignKeyModelTranslationOptions(SetNullForeignKeyBaseModelTranslationOptions, TranslationOptions):
```

Comment:
No need to inherit previous class:

![image](https://github.com/deschler/django-modeltranslation/assets/301015/8f31f956-e203-4916-9b7f-4a8c08853137)

Reply:
But my base class is abstract?

Reply:
Im my experience, it doesn't matter if the abstract or concrete model is registered. The problem is, that django thinks that the two fields of the child models belong to the same model, and then trigger the `TypeError / Cannot combine queries on two different base models` error, during the update query for the "models.SET_NULL" foreign keys. That's what I think, I might be wrong.

Reply:
Oh, i missed that part about abstract model. But anyway, you should not inherit options classes.

Reply:
But that's what docs do/say? https://django-modeltranslation.readthedocs.io/en/latest/registration.html#translationoptions-fields-inheritance

Abstract base class: Inherit TranslationOptionsBaseClass. Concrete base class: Don't inherit TranslationOptions.

For me, it doesn't matter, and I will happily do what solves the issue (as you said, translate/register on the model where the field is defined), but for now there is no solution, or I did not proceed correctly. If you run tests, you'll see yourself.

Reply:
And: Thank you for your help @last-partizan really appreciate!